packages/manager/modules/support/src/support.controller.js

```diff
@@ -0,0 +1,30 @@
+import { EVENT_NAMES } from './support.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $scope,
+  ) {
+    this.$scope = $scope;
+  }
+
+  $onInit() {
+    this.$scope.$on(
+      EVENT_NAMES.startLoading,
+      () => this.startLoading(),
+    );
+
+    this.$scope.$on(
+      EVENT_NAMES.stopLoading,
+      () => this.stopLoading(),
+    );
+  }
+
+  startLoading() {
+    this.isLoading = true;
```

Comment:
Imo, we should uniformize and use `ng-ui-router-line-progress` instead

Reply:
This is used for form submission, not for state transition, this is what was advised by CX :)